src/cbc_sdk/platform/audit.py

```diff
@@ -382,3 +385,27 @@ def _run_async_query(self, context):
         result = resp.json()
         results = result.get("results", [])
         return [self._doc_class(self._cb, item) for item in results]
+
+    def export(self, format="csv"):
```

Comment:
An example please so it's easy to use.
audit_log_query = cb.select(AuditLog).add_time_criteria(range="-1d")
audit_log_export_job = audit_log_query.export(format="csv")
results = audit_log_export_job.await_completion().result()

Reply:
Thanks for providing the example text. Added.